homeassistant/components/google/config_flow.py

```diff
@@ -162,3 +178,39 @@ async def async_step_reauth_confirm(
         if user_input is None:
             return self.async_show_form(step_id="reauth_confirm")
         return await self.async_step_user()
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Create an options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Google Calendar options flow."""
+
+    def __init__(self, config_entry):
```

Comment:
Please add type annotations so we keep the module typed.

Reply:
Maybe we can add google to the strictly typed list?

Reply:
Good catch, i didn't notice it wasn't enabled.

I have a couple things to fix in the upstream library so i've added a temporary ignore and put it in the strictly typed list.